scripts/sign_module

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "${SCRIPTS_DIR}/common"
+
+secret_key="./examples/certs/minisign/minisign.key"
+public_key="./examples/certs/minisign/minisign.pub"
+while getopts "i:o:s:p:h" opt; do
+  case "${opt}" in
+    h)
+      echo -e "Usage: ${0} [-h] -i INPUT_FILE -o OUTPUT_FILE
+
+Sign Wasm module using Ed25519 scheme.
+
+Options:
+  -i    Input Wasm file
+  -o    Output signature file
+  -h    Print Help (this message) and exit"
+      exit 0;;
+    i)
+      readonly INPUT_FILE="${OPTARG}";;
+    o)
+      readonly OUTPUT_FILE="${OPTARG}";;
```

Comment:
FWIW this script seems already complex enough that it may be easier to replace it with an actual small Rust binary, and then we also get the benefit that it can just generate self-contained signature files (including the hash, the public key, and the signature, in a single file). Initially it can literally spawn out to `minisign` if needed, or use the `minisign` crate if it works and can rely on the existing private key. Could you create a PR to track?